ArmoniK.SDK.Worker.Test/include/StressTest.h

```diff
@@ -0,0 +1,96 @@
+//
+// Created by fdenef on 01/02/2024.
+//
+
+#pragma once
+
+#include <algorithm>
+#include <armonik/sdk/worker/ServiceBase.h>
+#include <chrono>
+#include <cstring>
+#include <iostream>
+#include <numeric>
+#include <vector>
+
+namespace ArmoniK {
+namespace Sdk {
+namespace Worker {
+namespace Test {
+
+class StressTest final : public ServiceBase {
+public:
+  void *enter_session(const char *session_id) final {
+    std::cout << "StressTest enter session: " << session_id << '\n';
+    return new std::string(session_id);
+  }
+  void leave_session(void *session_ctx) final {
+    const auto session_id = static_cast<std::string *>(session_ctx);
+    std::cout << "StressTest leaving session: " << session_id << '\n';
+    delete session_id;
+  }
+  std::string call(void *, const std::string &name, const std::string &input) final {
+    if (name == "compute_workload") {
+      const auto resultWorkload = [&]() -> std::vector<double> {
+        const std::size_t sizeofNbOutput = sizeof(std::uint32_t);
+        std::uint32_t nbOutputBytes = 0;
+        std::uint32_t workLoadTimeInMs = 0;
+        std::uint32_t nbInputBytes = 0;
+
+        auto beginPtr = input.data();
+        std::memcpy(&nbOutputBytes, beginPtr, sizeofNbOutput);
+
+        beginPtr += sizeofNbOutput;
+        std::memcpy(&workLoadTimeInMs, beginPtr, sizeofNbOutput);
+
+        beginPtr += sizeofNbOutput;
+        std::memcpy(&nbInputBytes, beginPtr, sizeofNbOutput);
+
+        beginPtr += sizeofNbOutput;
+        std::vector<double> inputWorkload(nbInputBytes, 0.0);
+        std::memcpy(inputWorkload.data(), beginPtr, nbInputBytes * sizeof(double));
```

Comment:
```suggestion
        std::memcpy(inputWorkload.data(), beginPtr, nbInputBytes);
```
Found the problem !
You already sent the number of bytes not the number of elements.